facia/app/controllers/FaciaController.scala

```diff
@@ -190,17 +190,25 @@ trait FaciaController extends BaseController with Logging with ImplicitControlle
     }
   }
 
+  private def withFromShowMore(faciaCardAndIndex: FaciaCardAndIndex): FaciaCardAndIndex = faciaCardAndIndex.item match {
+    case contentCard: ContentCard => faciaCardAndIndex.copy(item = contentCard.copy(fromShowMore = true))
+    case _ => faciaCardAndIndex
+  }
```

Comment:
Can this be moved in `FaciaCardAndIndex` class?

Reply:
yes!